lom/src/plugins/sonic/plugin_integration_tests/detection/link/linkcrc/LINK_CRC_COUNTERS1.txt

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Suggestion only:
You can pout all this in one JSON file. A single data file, instead of many.
Being a test code -- this is *not* required. But better in longer run.

Reply:
Will keep it multiple files for now.